.gitignore

```diff
@@ -1,5 +1,12 @@
+# Ignore local rust builds
 /target
+
+# Ignore artifacts produced by local development
 config.json
+db/
```

Comment:
This is a directory that the task cache produces.  Unlike `keys/` where the directory has to be present upon initialization, this directory is created if it does not already exist.